test_util/summary.cpp

```diff
@@ -99,7 +99,11 @@ int main(int argc, char **argv) {
         }
     }
 
-    int argOffset = optind;
+    const int argOffset = optind;
+    if (argOffset > argc - 1) {
+        printHelp();
```

Comment:
Shouldn't we print a message that arguments are missing and return EXIT_FAILURE?

Reply:
The help message states this, and the other help message invocation already returned success. Is there a standard for what a program should return when printing the usage/help statement?

Reply:
I see your point, and have changed to return failure.